cleanlab_studio/internal/util.py

```diff
@@ -63,3 +64,130 @@ def check_none(x: Any) -> bool:
 
 def check_not_none(x: Any) -> bool:
     return not check_none(x)
+
+
+def _get_autofix_default_params() -> dict:
+    """returns default percentage-wise params of autofix"""
+    return {
+        "ambiguous": 0.2,
+        "label_issue": 0.5,
+        "near_duplicate": 0.2,
+        "outlier": 0.5,
+        "confidence_threshold": 0.95,
+    }
+
+
+def _get_autofix_defaults(cleanset_df: pd.DataFrame) -> dict:
+    """
+    Generate default values for autofix parameters based on the size of the cleaned dataset.
+    """
+    default_params = _get_autofix_default_params()
+    default_values = {}
+
+    for param_name, param_value in default_params.items():
+        if param_name != "confidence_threshold":
+            num_rows = cleanset_df[f"is_{param_name}"].sum()
+            default_values[f"drop_{param_name}"] = math.ceil(num_rows * param_value)
+        else:
+            default_values[f"drop_{param_name}"] = param_value
+    return default_values
+
+
+def _get_top_fraction_ids(
+    cleanset_df: pd.DataFrame, name_col: str, num_rows: int, asc=True
+) -> List[str]:
+    """
+    Extracts the top specified number of rows based on a specified score column from a DataFrame.
+
+    Parameters:
+    - cleanset_df (pd.DataFrame): The input DataFrame containing the cleanset.
+    - name_col (str): The name of the column indicating the category for which the top rows should be extracted.
+    - num_rows (int): The number of rows to be extracted.
```

Comment:
In autofix, we can simply multiply the fraction of issues that are the cleanset defaults by the number of datapoints to get this.

Reply:
right when we spoke originally, we wanted this call to be similar to the Studio web interface call, hence I rewrote it this way, it was floating percentage before. 
the function `_get_autofix_defaults` does the multiplication by number of datapoints